src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/iceberg/IcebergFieldNode.java

```diff
@@ -61,26 +120,23 @@ StringBuilder buildQuery(StringBuilder sb, String parentType) {
     return sb;
   }
 
-  private StringBuilder appendNameAndType(StringBuilder sb) {
```

Comment:
Let's separate SQL generation to a separate class. Node class is having a lot of responsibilities right now.

Reply:
agree

Reply:
done